test/classifier_sequence_test.py

```diff
@@ -0,0 +1,68 @@
+import collections
+import pytest
+from test import icon_builder as ib
```

Comment:
You use `ib` in exactly one place, so I think the alias to shorten the name is not needed and just leaving it as `icon_builder` is clearer.

Reply:
I changed it to ib because it was conflicting with the fixture with the same name.